Measurements/Tasks.qs

```diff
@@ -68,21 +68,21 @@ namespace Quantum.Kata.Measurements {
     
     // Task 1.4. |A⟩ or |B⟩ ?
     // Inputs:
-    //      1) angle alpha, in radians, represented as Double
+    //      1) angle α, in radians, represented as a Double
     //      2) a qubit which is guaranteed to be in |A⟩ or |B⟩ state
-    //         |A⟩ =   cos(alpha) * |0⟩ + sin(alpha) * |1⟩,
-    //         |B⟩ = - sin(alpha) * |0⟩ + cos(alpha) * |1⟩.
-    // Output: true if qubit was in |A⟩ state, or false if it was in |B⟩ state.
+    //         |A⟩ =   cos α |0⟩ + sin α |1⟩,
+    //         |B⟩ = - sin α |0⟩ + cos α |1⟩.
+    // Output: true if the qubit was in |A⟩ state, or false if it was in |B⟩ state.
     // The state of the qubit at the end of the operation does not matter.
-    operation IsQubitA (alpha : Double, q : Qubit) : Bool {
+    operation IsQubitA (α : Double, q : Qubit) : Bool {
```

Comment:
I like α, but it might annoy some people who can't figure out how to type it and don't think of using copy and paste...